rs/nns/cmc/cmc.did

```diff
@@ -5,14 +5,13 @@ type log_visibility = variant {
   public;
 };
 type CanisterSettings = record {
-  controller : opt principal;
```

Comment:
This is the only functional change (the rest is formatting, seems like the file was not formatted for a long time). Removing this field should be fine from a Candid compatibility pov -- unknown fields will be ignored when decoding.